new-system-tests/src/test/java/io/apicurio/registry/systemtests/Utils.java

```diff
@@ -0,0 +1,139 @@
+package io.apicurio.registry.systemtests;
+
+import io.apicur.registry.v1.ApicurioRegistry;
+import io.apicur.registry.v1.ApicurioRegistryBuilder;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.client.KubernetesClientTimeoutException;
+import io.fabric8.openshift.client.OpenShiftClient;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Utilities used in test suite and tests.
+ */
+public class Utils {
+    /**
+     * Finds {@link Deployment} of Apicurio Registry operator in provided namespace.
+     *
+     * @param client {@link OpenShiftClient} instance to use for interaction with cluster.
+     * @param namespace Name of namespace to search for Apicurio Registry operator {@link Deployment}.
+     * @return {@link Deployment} of Apicurio Registry operator if {@link Deployment} was found; {@code null} otherwise.
+     */
+    public static Deployment findRegistryOperatorDeployment(OpenShiftClient client, String namespace) {
+        return client
+                .apps()
+                .deployments()
+                .inNamespace(namespace)
+                .list()
+                .getItems()
+                .stream()
+                .filter(d -> d.getMetadata().getName().startsWith(Constants.REGISTRY_OPERATOR_NAME))
```

Comment:
There is a difference between community Apicurio Registry operator deployment name `apicurio-registry-operator` and current Red Hat Integration Service Registry operator deployment name `apicurio-registry-operator-v1.1.0-redhat.1`, so I use prefix instead of exact name match.

Reply:
fair 👍